Mathlib/FieldTheory/IsAlgClosed/Basic.lean

```diff
@@ -64,22 +64,22 @@ class IsAlgClosed : Prop where
 See also `IsAlgClosed.splits_domain` for the case where `K` is algebraically closed.
 -/
 theorem IsAlgClosed.splits_codomain {k K : Type*} [Field k] [IsAlgClosed k] [CommRing K]
-    {f : K →+* k} (p : K[X]) : p.Splits f :=
+    {f : K →+* k} (p : K[X]) : (p.map f).Splits :=
   IsAlgClosed.factors (p.map f)
 
 /-- Every polynomial splits in the field extension `f : K →+* k` if `K` is algebraically closed.
 
 See also `IsAlgClosed.splits_codomain` for the case where `k` is algebraically closed.
 -/
 theorem IsAlgClosed.splits_domain {k K : Type*} [Field k] [IsAlgClosed k] [Field K] {f : k →+* K}
-    (p : k[X]) : p.Splits f :=
+    (p : k[X]) : (p.map f).Splits :=
   (IsAlgClosed.factors p).map f
```

Comment:
These two can be deprecated and replaced with `splits`/`factors` + `Splits.map`.